crates/cxx-qt-gen/src/generator/rust/property/getter.rs

```diff
@@ -43,7 +43,7 @@ pub fn generate(
                     #[doc = "Getter for the Q_PROPERTY "]
                     #[doc = #ident_str]
                     pub fn #getter_rust(&self) -> &#ty {
-                        &self.rust().#ident
```

Comment:
Hm, it seems like these getters have become kind of pointless in Rust at least.
We do still use them in C++ though, so maybe something to think about removing later.

Reply:
Yup, but we still need something to bind the C++ to for it's getter, as we can't use member in C++ etc.

Interestingly we can remove the "wrapper" getter we have via the same solution as #589 if it's accepted upstream :-)